### DISCUSSION TITLE
What is n_signal and why does it defualt to 131072?

### DISCUSSION THREAD

**Discussion by alex-frx** [acids-ircam/RAVE#179] [Q&A]
Like asked abov what is n_signal what does the changing of that number change?

**Comment by jacklion710** [comment]
I believe your answer is [here](https://github.com/acids-ircam/RAVE/discussions/61)

"_As far as I know RAVE will chop up your audio files into chunks of size N_SIGNAL during training. The default value for N_SIGNAL is 65536, so about 1.36 seconds. My guess is that this should be your minimal audio file size..._"
